src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/InternalApi.cs

```diff
@@ -317,13 +317,16 @@ public static void SetApplicationName(string applicationName, string? applicatio
             return _agentApiImplementation?.GetResponseMetadata() ?? Enumerable.Empty<KeyValuePair<string, string>>();
         }
 
-        /// <summary>
-        /// DOCS GO HERE
+        /// <summary> Sets the method that will be invoked to define the error group that an exception
+        /// should belong to.
+        ///
+        /// The callback takes an Exception and returns the name of the error group to use. Return values
+        /// that are null, empty, or whitespace will not associate the Exception to an error group.
         /// </summary>
-        /// <param name="callback"></param>
+        /// <param name="callback">The callback to invoke to define the error group that an Exception belongs to.</param>
         public static void ErrorFingerprintingCallback(Func<Exception, string> callback)
```

Comment:
We should probably rename this to match the others?

Reply:
Yes, good catch. I'll do a search to see if there are others that need renamed.

Reply:
After c29f60f the remaining references to the fingerprint callback are in the usage of the callback, which should be updated in #1401.